pdf/plugin-sankey/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Sankey Plot</title>
+    <style>
+        svg {
+            margin-block: 1em;
+        }
+    </style>
+</head>
+<body>
+    <p>Warning: Experimental. Does not work on malformed PDFs.</p>
+    <details>
+        <summary>Debug Info</summary>
+        <ul>
+            {% for line in DEBUG_LINES %}
+                <li><pre>{{ line }}</pre></li>
+            {% endfor %}
+        </ul>
+    </details>
+    <label>Vertical Zoom: <input type="number" id="height-selector" pattern="[0-9]+" title="Vertical Zoom"></label><br/>
+    <script>
+        const sankeyCsvText = {{ CSV_TEXT }};
+    </script>
+    <!-- TODO would be better to download these in a build stage -->
```

Comment:
Whoops, forgot to update this before making the PR. It's worth doing before merging IMO. Though unfortunately I think the canonical place for both these sources is npm

Reply:
I like the static path idea! Re:disambiguation, there's already no protection against plugins/parsers having name collisions in their existing build output, so imo a really robust solution wouldn't be worth it. Could just establish a convention of having a subpath for each plugin, specified explicitly in the build stage and any of the plugin's html files. This would also allow sharing files between plugins, which could help in some cases.

Reply:
I added an implementation of this to this branch. Could use some scrutiny before merging since it does have broader effects, but it's a fairly small change.